### PR TITLE
Add `ZINTER/ZUNION` command

### PR DESCRIPTION
**Pull request by yangbodong22011**
see: https://github.com/redis/redis/pull/7794

### CODE REVIEW

File: commands/zinter.md
Comment (oranagra) [inline commands/zinter.md:6]: shouldn't this have something similar to ZRANGE?
```
@array-reply: list of elements in the specified range (optionally with
their scores, in case the `WITHSCORES` option is given).
```

Comment (oranagra) [reply commands/zinter.md:6]: ohh, my bad. it has it.

**Review by oranagra — APPROVED** [review]

**Comment by oranagra** [comment]
@itamarhaber remind me, do we merge these once the command was merged to unstable? or wait for a release?

File: commands/zinter.md
Comment (itamarhaber) [inline commands/zinter.md:2]:
```suggestion
This command is similar to `ZINTERSTORE`, but instead of storing the resulting
sorted set, it is returned to the client.
```

File: commands/zunion.md
Comment (itamarhaber) [inline commands/zunion.md:2]:
```suggestion
This command is similar to `ZUNIONSTORE`, but instead of storing the resulting
sorted set, it is returned to the client.
```

**Review by oranagra — APPROVED** [review]

**Pull request merged**
